message/outbound_msg_builder.go

```diff
@@ -560,17 +563,19 @@ func (b *outMsgBuilder) PushQuery(
 	requestID uint32,
 	deadline time.Duration,
 	container []byte,
+	requestedHeight uint64,
 	engineType p2p.EngineType,
 ) (OutboundMessage, error) {
 	return b.builder.createOutbound(
 		&p2p.Message{
 			Message: &p2p.Message_PushQuery{
 				PushQuery: &p2p.PushQuery{
-					ChainId:    chainID[:],
-					RequestId:  requestID,
-					Deadline:   uint64(deadline),
-					Container:  container,
-					EngineType: engineType,
+					ChainId:         chainID[:],
+					RequestId:       requestID,
+					Deadline:        uint64(deadline),
+					Container:       container,
+					EngineType:      engineType,
+					RequestedHeight: requestedHeight,
```

Comment:
super nit... (to be consistent with arg order :) )

```suggestion
					RequestedHeight: requestedHeight,
					EngineType:      engineType,
```

Reply:
Agree we should unify with InboundMsgBuilder